src/panels/lovelace/common/compute-card-size.ts

```diff
@@ -1,5 +1,12 @@
 import { LovelaceCard } from "../types";
 
-export const computeCardSize = (card: LovelaceCard): number => {
-  return typeof card.getCardSize === "function" ? card.getCardSize() : 4;
+export const computeCardSize = async (card: LovelaceCard): Promise<number> => {
```

Comment:
I suggest that we don't make this function async, so that we can avoid an unnecessary await. 
```suggestion
export const computeCardSize = (card: LovelaceCard): number | Promise<number> => {
```